db/structure.sql

```diff
@@ -592,8 +593,8 @@ ALTER SEQUENCE public.pom_details_id_seq OWNED BY public.pom_details.id;
 --
 
 CREATE TABLE public.prisons (
-    code character varying NOT NULL,
     prison_type character varying NOT NULL,
+    code character varying NOT NULL,
```

Comment:
Is this a change that was missed from a previous migration?

Reply:
😮

Reply:
Ah, I see what happened - it's rearranged itself.